### PR TITLE
Add wstETH as collateral to USDC market on Base

### PR DESCRIPTION
**Pull request by MishaShWoof**

### CODE REVIEW

File: deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        8,                               // decimals
+        'wstETH / ETH price feed'        // description
+      ]
+    );
+    const _wstETHScalingPriceFeed = await deploymentManager.deploy(
+      'wstETH:priceFeed',
+      'pricefeeds/MultiplicativePriceFeed.sol',
+      [
+        _wstETHToETHPriceFeed.address,                                                 // wstETH / ETH price feed
+        (await deploymentManager.fromDep('WETH:priceFeed', 'base', 'usdc')).address,   // ETH / USD price feed
```
Comment (dmitriy-woof-software) [inline deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts:29]: receive the value before `const _wstETHScalingPriceFeed`

File: deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      'wstETH:priceFeed to ETH',
+      'pricefeeds/MultiplicativePriceFeed.sol',
+      [
+        WSTETH_STETH_PRICE_FEED_ADDRESS, // wstETH / stETH price feed
+        STETH_ETH_PRICE_FEED_ADDRESS,    // stETH / ETH price feed
+        8,                               // decimals
+        'wstETH / ETH price feed'        // description
+      ]
+    );
+    const _wstETHScalingPriceFeed = await deploymentManager.deploy(
```
Comment (dmitriy-woof-software) [inline deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts:24]: 4th argument of deploy() is `true`

File: deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    const _wstETHToETHPriceFeed = await deploymentManager.deploy(
+      'wstETH:priceFeed',
+      'pricefeeds/MultiplicativePriceFeed.sol',
+      [
+        WSTETH_STETH_PRICE_FEED_ADDRESS, // wstETH / stETH price feed
+        STETH_ETH_PRICE_FEED_ADDRESS,    // stETH / ETH price feed
+        8,                               // decimals
+        'wstETH / ETH price feed'        // description
+      ],
+      true
```
Comment (dmitriy-woof-software) [inline deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts:23]: Remove `true`

File: deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    const _wstETHScalingPriceFeed = await deploymentManager.deploy(
+      'wstETH:priceFeed',
+      'pricefeeds/MultiplicativePriceFeed.sol',
+      [
+        _wstETHToETHPriceFeed.address, // wstETH / ETH price feed
+        _ETHToUSDPriceFeed.address,    // ETH / USD price feed
+        8,                             // decimals
+        'wstETH / USD price feed'      // description
+      ]
+    );
```
Comment (dmitriy-woof-software) [inline deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts:35]: add `true` here

File: deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts (hunk 3 above)
Comment (torreyatcitty) [reply deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts]: wait, did we need to do this still?

File: deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { calldata, exp, proposal } from '../../../../src/deploy';
+import { utils } from 'ethers';
+
+const WSTETH_ADDRESS = '0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452';
+const WSTETH_STETH_PRICE_FEED_ADDRESS = '0xB88BAc61a4Ca37C43a3725912B1f472c9A5bc061';
+const STETH_ETH_PRICE_FEED_ADDRESS = '0xf586d0728a47229e747d824a939000Cf21dEF5A0';
+let newPriceFeedAddress: string;
+
+export default migration('1721034240_add_wsteth_as_collateral.ts', {
```
[the remark this answers is not in the document]
Comment (torreyatcitty) [reply deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts]: wait, did we need to do this still?

[the remark this answers is not in the document]
Comment (dmitriy-woof-software) [reply deployments/base/usdc/migrations/1721034240_add_wsteth_as_collateral.ts]: resolved

**Review by torreyatcitty — APPROVED** [review]
LGTM

**Pull request merged**